packages/yoga/src/Popover/web/Popover.jsx

```diff
@@ -49,16 +69,11 @@ function Popover({
 
       <PopoverButton
         {...(a11yId && { 'aria-describedby': a11yId })}
-        onMouseEnter={handleShowPopover}
-        onMouseLeave={handleHidePopover}
-        onTouchStart={handleShowPopover}
-        onTouchEnd={handleHidePopover}
-        onClick={event => event.preventDefault()}
-        onKeyDown={event => {
-          if (event.key === 'Enter') {
-            setIsPopoverOpen(current => !current);
-          }
-        }}
+        ref={ref}
+        onMouseEnter={hover ? handleShowPopover : undefined}
+        onMouseLeave={hover ? handleHidePopover : undefined}
+        onClick={!hover ? toggleOpenPopover : undefined}
```

Comment:
As discussed offline, previous behaviour should be preserved:

https://github.com/gympass/yoga/pull/643